config/production.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```

Comment:
just an example, right? think you can remove it if it doesn't error for being empty!